### PR TITLE
added instances, sec-group, loadblanacer

### PR DESCRIPTION
**Pull request by HelgaStock**

### CODE REVIEW

**Comment by HelgaStock** [comment]
update: ec2.tf was in wrong folder, over to folder architecture.

**Comment by fabianschmauder** [comment]
maybe give me a hint when you need a new review. We discussed the stuff already in the 1on1

File: architecture/terraform.tfstate [hunk 1]
```diff
@@ -0,0 +1,851 @@
+{
```
Comment (fabianschmauder) [inline architecture/terraform.tfstate:1]: do not upload your tf state file. It could include sesnstive informations about your system

Comment (fabianschmauder) [reply architecture/terraform.tfstate:1]: better use s3 as backend config for your terraform state

File: architecture/terraform.tfstate.backup [hunk 2]
```diff
@@ -0,0 +1,841 @@
+{
```
Comment (fabianschmauder) [inline architecture/terraform.tfstate.backup:1]: do not upload this file

**Review by fabianschmauder — COMMENTED** [review]
Remove the state files. Rest looks fine

**Review by HelgaStock — COMMENTED** [review]
testate files removed, ready to merge

**Pull request merged**